### PR TITLE
primary key can be changed if index is empty

### LINKED ISSUE #1574: primary key can be changed if index is empty
As discussed in this [slack message](https://meilisearch.slack.com/archives/C01K0CZKASZ/p1649061495862959) (private link) and this [PR comment](https://github.com/meilisearch/specifications/pull/132/files#r836682649), it is now possible to change the primary key of an empty index using the [update index endpoint](https://docs.meilisearch.com/reference/api/indexes.html#update-an-index).

Before, you could only do this if the index had never possessed a primary key.

I'm not sure which pages are affected by this change, but certainly the update index endpoint (linked above) could be clarified, and possibly also [the primary key guide](https://github.com/meilisearch/documentation/pull/1544).

### PR DESCRIPTION
**Pull request by maryamsulemani97**
closes #1574

### CODE REVIEW

File: reference/api/indexes.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ Update an [index's](/learn/core_concepts/indexes.md) [primary key](/learn/core_c
 
 If a primary key wasn't explicitly chosen during index creation, you can use this route to configure it. If you are unsure whether an index's primary key has already been configured, you can use the [get index endpoint](/reference/api/indexes.md#get-one-index) to verify it.
 
+You can update the primary key of an empty index. If your index contains documents, you will need to delete all of them first, then change the primary key, and add them again.
```
Comment (guimachiavelli) [inline reference/api/indexes.md:121]: At first I just wanted to separate empty/non-empty cases, but then the editing bug got me. Is this better?

```suggestion
You can freely update the primary key of an empty index.

You can also change the primary key of an index that already contains documents. To do that, you must first delete all documents in that index. You must then change the primary key, and finally index your dataset again.
```

File: reference/api/indexes.md [hunk 2]
```diff
@@ -118,6 +118,10 @@ Update an [index's](/learn/core_concepts/indexes.md) [primary key](/learn/core_c
 
 If a primary key wasn't explicitly chosen during index creation, you can use this route to configure it. If you are unsure whether an index's primary key has already been configured, you can use the [get index endpoint](/reference/api/indexes.md#get-one-index) to verify it.
 
+You can freely update the primary key of an empty index.
```
Comment (dichotommy) [inline reference/api/indexes.md:121]:
```suggestion
You can freely update the primary key of an index as long as it contains no documents.
```

File: reference/api/indexes.md [hunk 3]
```diff
@@ -118,6 +118,10 @@ Update an [index's](/learn/core_concepts/indexes.md) [primary key](/learn/core_c
...
+
+You can also change the primary key of an index that already contains documents. To do that, you must first delete all documents in that index. You must then change the primary key, and finally index your dataset again.
```
Comment (dichotommy) [inline reference/api/indexes.md:123]:
```suggestion
To change the primary key of an index that already contains documents, you must first delete all documents in that index. You may then change the primary key and, finally, index your dataset again.
```

File: reference/api/indexes.md [hunk 4]
```diff
@@ -118,6 +118,10 @@ Update an [index's](/learn/core_concepts/indexes.md) [primary key](/learn/core_c
 
 If a primary key wasn't explicitly chosen during index creation, you can use this route to configure it. If you are unsure whether an index's primary key has already been configured, you can use the [get index endpoint](/reference/api/indexes.md#get-one-index) to verify it.
```
Comment (dichotommy) [inline reference/api/indexes.md:119]:
```suggestion
```

**Review by dichotommy — CHANGES_REQUESTED** [review]

**Review by dichotommy — APPROVED** [review]
LGTM! 🧞

**Comment by maryamsulemani97** [comment]
bors merge

**Pull request merged**